resharper/resharper-unity/src/UnityInstallationFinder.cs

```diff
@@ -23,16 +23,16 @@ public FileSystemPath GetApplicationPath(Version version)
         {
             var possible = GetPossibleInstallationInfos();
             
-            var bestChoice = possible.FirstOrDefault(a =>
+            var bestChoice = possible.LastOrDefault(a =>
```

Comment:
I don't think `GetPossibleInstallationInfos` has a deterministic order. It looks like the directories are ordered by full path, rather than version, and then when the install info is created, it's not sorted again.

Reply:
Ah, it may be a small problem, I will resort then by version again.